docs/install_guide.adoc

```diff
@@ -175,6 +175,86 @@ $ oc describe secret osp-secret -n openstack | grep Watcher
 WatcherPassword:                  9 bytes
 ----
 +
+. Watcher does not support creating certificates for TLS, but assumes that the
+user has created them beforehand. To create the certificates, use the Cert-Manager Operator
+installed by the Openstack Operator and create `Certificate` objects. For TLS
+everywhere, three Certificates are needed, one for the route, one for the
+internal endpoint and one for the public endpoint.
++
+Before creating the certificates, note the project where the openstack
+controlplane is deployed, which by default is __openstack__. When creating the
+certificates, you will need to set the name of the project in the DNS names
+section. Create a `watcher-certificates.yaml` file:
```

Comment:
I'd make more clear that user will need to replace `example.com` in the watcher-public-route Certificate CR by the route domain used on the user deployment. The default domain can be found with:

$ oc get ingresses.config/cluster -o jsonpath={.spec.domain}

Reply:
thanks, I did not know about that command, added it